src/context/MapContext.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { createContext, useState } from 'react'
+
+interface MapContextType {
+  mapApi: kakao.maps.Map | null
+  markers: kakao.maps.Marker[]
+  setMapApi: (newMap: kakao.maps.Map) => void
+  setMarkers: (newMarker: kakao.maps.Marker) => void
+  deleteMarkers: () => void
+}
+
+export const MapContext = createContext<MapContextType>({
+  mapApi: null,
+  markers: [],
+  setMapApi: (newMap) => {
+    return null
+  },

```

Comment:
(p4) void 리턴형이라 return null 할 필요는 없을 것 같아요
혹은 아래와 같이 축약할 수도 있습니다
```suggestion
  setMapApi: () => {},
```